.github/workflows/osrm-backend.yml

```diff
@@ -34,12 +36,38 @@ jobs:
         node-version: 16
```

Comment:
Using Node v16, not sure if we need matrix build for Windows building for older versions.

Reply:
We may need Node v18 though too(which is currently latest)